### PR TITLE
Push net-exporter to default app catalog

### PR DESCRIPTION
**Pull request by sslavic**
Towards giantswarm/giantswarm#7029, specifically migrating core components to App Catalog.

### CODE REVIEW

File: helm/net-exporter/values.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: net-exporter
+namespace: monitoring
```
Comment (rossf7) [inline helm/net-exporter/values.yaml]: Damn this has the same namespace problem as cert-exporter.

What about we change the value to `.exporter.namespace` and then set it to `kube-system` in the default app catalog?

Comment (MarcelMue) [reply helm/net-exporter/values.yaml]: +1000 this should not end up in the `monitoring` namespace in tenant clusters. That will be very confusing.

Comment (tomahawk28) [reply helm/net-exporter/values.yaml]: should it belong to `kube-system` or `giantswarm` in TC? I'm confusing as well.

Comment (rossf7) [reply helm/net-exporter/values.yaml]: `kube-system` same as it is now. `giantswarm` namespace is for chart-operator, tiler and related resources.

Comment (rossf7) [reply helm/net-exporter/values.yaml]: See https://github.com/giantswarm/cert-exporter/pull/44#discussion_r337879908 

For tenant clusters we create a configmap that sets the namespace to `kube-system`.

In the control plane we use the `monitoring` namespace that is the default value. We need the equivalent of this logic for the new chart.

File: helm/net-exporter/templates/np.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    to:
+    - podSelector:
+        matchLabels:
+          app: {{ .Values.name }}
+  - ports:
+    - port: {{ .Values.dns.port }}
+      protocol: UDP
+    - port: {{ .Values.dns.port }}
+      protocol: TCP
+    # legacy part kept for compatibility
```
Comment (rossf7) [inline helm/net-exporter/templates/np.yaml]:
```suggestion
    # legacy port kept for compatibility
```

This triggered me :)

File: helm/net-exporter/templates/np.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    - podSelector:
+        matchLabels:
+          k8s-app: {{ .Values.dns.label }}
+      namespaceSelector:
+        matchLabels:
+          name: kube-system
+  - ports:
+    - port: 443
+      protocol: TCP
+    # legacy part kept for compatibility
```
Comment (rossf7) [inline helm/net-exporter/templates/np.yaml]:
```suggestion
    # legacy port kept for compatibility
```

**Pull request merged**